src/lib/AnnotationModule.ts

```diff
@@ -0,0 +1,40 @@
+import {
+    bdlBoxBlue,
+    bdlGreenLight,
+    bdlWatermelonRed,
+    bdlYellorange,
+    bdlYellow,
+    bdlGrimace,
+} from 'box-ui-elements/es/styles/variables';
+import Cache from './Cache';
+
+export const AnnotationColor = {
+    BOX_BLUE: bdlBoxBlue,
+    GREEN_LIGHT: bdlGreenLight,
+    WATERMELON_RED: bdlWatermelonRed,
+    YELLORANGE: bdlYellorange,
+    YELLOW: bdlYellow,
+    GRIMACE: bdlGrimace,
+};
+
+type Options = {
+    cache: Cache;
+};
+
+const ANNOTATION_COLOR_KEY = 'annotation-color';
+
+export default class AnnotationModule {
+    private cache: Cache;
+
+    constructor(options: Options) {
+        this.cache = options.cache;
```

Comment:
We can destructure `cache` in the constructor signature.

Reply:
Will do.